src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxResolvingClassLoader.java

```diff
@@ -89,8 +97,12 @@
         }
     }
 
-    private static <T> LoadingCache<ClassLoader, Cache<String, T>> makeParentCache() {
-        return CacheBuilder.newBuilder().weakKeys().build(new CacheLoader<ClassLoader, Cache<String, T>>() {
+    private static <T> LoadingCache<ClassLoader, Cache<String, T>> makeParentCache(boolean weakValues) {
+        CacheBuilder<Object, Object> builder = CacheBuilder.newBuilder().weakKeys();
```

Comment:
Will again be a merge conflict with #160.

Reply:
Easy fix.

Reply:
Yes (assuming Caffeine also support `weakValues`), just FYI so it does not come as a surprise.

Reply:
It does.